CMakeLists_files.cmake

```diff
@@ -31,6 +31,10 @@
 list (APPEND TEST_SOURCE_FILES
 	)
 
+list(APPEND MAIN_SOURCE_FILES
+	opm/upscaling/RelPermUtils.cpp
```

Comment:
they are private headers. i'm not aware of any such consistent way 
currently. i'll happily adjust if i am wrong though, now or in the 
future. in fact the entire upscaling library consists of private app 
code, some shared across multiple applications, some just used in a 
single app. but currently there is no way to have an app consist of 
multiple sources in the build system, so a library is required.